bigtable/client/instance_admin.cc

```diff
@@ -107,6 +107,77 @@ google::bigtable::admin::v2::Instance InstanceAdmin::CreateInstanceImpl(
   return result;
 }
 
+std::future<google::bigtable::admin::v2::Instance>
+InstanceAdmin::UpdateInstance(google::bigtable::admin::v2::Instance* instance,
```

Comment:
Same comment about the pointer parameters.

Reply:
changed to `InstanceUpdateconfig`